doc/descriptors.md

```diff
@@ -131,6 +131,27 @@ Key order does not matter for `sortedmulti()`. `sortedmulti()` behaves in the sa
 as `multi()` does but the keys are reordered in the resulting script such that they
 are lexicographically ordered as described in BIP67.
 
+#### Basic multisig example
+
+For a good example of a basic M-of-N multisig between multiple participants using descriptor
+wallets and PSBTs, as well as a signing flow, see [this functional test](/test/functional/wallet_multisig_descriptor_psbt.py).
+The basic steps are:
+
+  1. Every participant generates an xpub. The most straightforward way is to create a new wallet
+  2. The multisig is created by importing the following descriptors: `wsh(sortedmulti(<M>,xpubA/<0,1>/*,xpubB/<0,1>/*,…,xpubN/<0,1>/*))`
+    (one descriptor w/ `0` for receiving addresses and another w/ `1` for change). The resulting
+    wallet is watch-only and every participant does this
+  3. A receiving address is generated for the multisig
+  4. Funds are sent to the resulting address (every participant checks that it gets the same addresses)
```

Comment:
`every participant checks that it gets the same addresses)` maybe move to step 3